### PR TITLE
Securisation routes utilisateur/responsable

### PR DESCRIPTION
**Pull request by elisematos**

### CODE REVIEW

**Comment by elisematos** [comment]
https://www.loom.com/share/5ae2cadfe45443bdb264c30a9623dd5f

**Review by Frutios — APPROVED** [review]

File: config/packages/security.yaml [hunk 1]
```diff
@@ -1,13 +1,31 @@
 security:
+    role_hierarchy:
+        ROLE_UTILISATEUR: ROLE_USER
+        ROLE_RESPONSABLE:  ROLE_UTILISATEUR
```
Comment (RomainClair) [inline config/packages/security.yaml]: What's this strange role hierarchy ?
do you really need a ROLE_UTILISATEUR and a ROLE_USER ?!?
You really should use english for your code.

**Review by RomainClair — CHANGES_REQUESTED** [review]
Your AppFixture is useless, why did you keep it ?

If you want to display "Vous êtes connecté en tant que xxx" on every page, why didn't you add this code ine the base template ?

File: config/packages/security.yaml (hunk 1 above)
Comment (elisematos) [reply config/packages/security.yaml]: Deleted AppFixtures.php which is indeed useless
Added "Vous êtes connnecté en tant que X" to base.html.twig
Renamed ROLES

**Review by RomainClair — APPROVED** [review]

**Pull request closed without merging**